pydoctor/templatewriter/writer.py

```diff
@@ -63,9 +64,9 @@ def writeDocsFor(self, ob, functionpages):
             if self.dry_run:
                 self.total_pages += 1
             else:
-                f = open(os.path.join(self.base, link(ob)), 'wb')
-                self.writeDocsForOne(ob, f)
-                f.close()
+                path = FilePath(self.base).child(f'{ob.fullName()}.html')
+                with path.open('wb') as f:
+                    self.writeDocsForOne(ob, f)
```

Comment:
```suggestion
                with path.open('wb') as stream:
                    self.writeDocsForOne(ob, stream)
```

I am new to py3 and I was a bit confused by the previous `f'{ob.fullName()}.html` 

I like the `f` literal ...but I think that is a 3.6 thing...and in twisted 3.5 is still supported ... but I am happy to remove support for 3.5 :)

Reply:
I'll use `out`: that's the name I typically use in new code, but here the old code was using `f` already so that stuck.

> I am new to py3 and I was a bit confused by the previous `f'{ob.fullName()}.html`
> 
> I like the `f` literal ...but I think that is a 3.6 thing...and in twisted 3.5 is still supported ... but I am happy to remove support for 3.5 :)

Yes, it's a 3.6 feature, which is fine for pydoctor but not yet for Twisted. I think the plan was to drop 3.5 in Twisted after the next release.